subscribers/order_events.go

```diff
@@ -24,7 +24,7 @@ import (
 
 type OE interface {
 	events.Event
-	Order() *types.Order
+	GetOrder() *types.Order
```

Comment:
I am probably missing something here, but why do you need this? The method on e.g. vega.events.order is 'Order' not GetOrder()

Reply:
golang won't permit me to add the Order() method as the embedded type is called Order in the OrderEventWithVegaTime, hence GetOrder